ingestion/tests/unit/topology/metadata/test_atlas.py

```diff
@@ -0,0 +1,612 @@
+                            ),
+                            Column(
+                                name="col4",
+                                displayName=None,
+                                dataType="BIGINT",
+                                arrayDataType=None,
+                                dataLength=None,
+                                precision=None,
+                                scale=None,
+                                dataTypeDisplay="bigint",
+                                description=None,
+                                fullyQualifiedName="delta.default.test_schema.delta_test_table.col2.col4",
+                                tags=None,
+                                constraint=None,
+                                ordinalPosition=None,
+                                jsonSchema=None,
+                                children=None,
+                                customMetrics=None,
+                                profile=None,
+                            ),
+                        ],
+                        customMetrics=None,
+                        profile=None,
+                    ),
+                    Column(
+                        name="col1",
+                        displayName=None,
+                        dataType="BIGINT",
+                        arrayDataType=None,
+                        dataLength=1,
+                        precision=None,
+                        scale=None,
+                        dataTypeDisplay="bigint",
+                        description="col1 description",
+                        fullyQualifiedName="delta.default.test_schema.delta_test_table.col1",
+                        tags=None,
+                        constraint=None,
+                        ordinalPosition=None,
+                        jsonSchema=None,
+                        children=None,
+                        customMetrics=None,
+                        profile=None,
+                    ),
+                ],
+                databaseSchema=EntityReference(
+                    id=mock_database_schema_object.id, type="databaseSchema"
+                ),
+            ),
+        )
+
+    def mock_ingest_lineage(self, source_guid, name):  # pylint: disable=unused-argument
+        return []
+
+    def mock_create_tag(self):
+        return []
+
+    @patch.object(AtlasClient, "list_entities", mock_list_entities)
+    @patch.object(AtlasClient, "get_entity", mock_get_entity)
+    @patch.object(AtlasSource, "ingest_lineage", mock_ingest_lineage)
+    @patch.object(AtlasSource, "create_tag", mock_create_tag)
+    def test_description(self):
+        """
+        Testing description updated for database, databaseSchema, table
+        """
+        _ = list(self.atlas_source.next_record())
+        updated_database = self.metadata.get_by_name(
+            entity=Database, fqn="hive.Reporting"
+        )
+        assert updated_database.description == EXPTECTED_DATABASE.description
```

Comment:
if we are just checking the description then it's okay just to assert it with string instead of creating the entire object and then just matching the description

Reply:
OK 👍